include/onnxruntime/core/graph/graph.h

```diff
@@ -625,6 +625,9 @@ class Graph {
   /** Check if a given name is an initializer tensor's name in this graph. */
   bool IsInitializedTensor(const std::string& name) const;
 
+  /** Check if a given name is a sparse initializer's name in the model */
+  bool IsSparseInitializer(const std::string& name) const;
```

Comment:
nit: IsSpareInitializedTensor for consistency with similar methods ?

Reply:
We currently convert all sparse initializers into dense Tensor. However, in some places we want to know if the initializer was actually sparse. This is a helper to query that.

Reply:
Sorry, maybe I wasn't clear. I was referring to the naming of the method. I understand the potential use-cases for this helper.

Reply:
Updated comments per our conversation.